pkg/sql/catalog/colinfo/result_columns.go

```diff
@@ -268,3 +268,14 @@ var TenantColumns = ResultColumns{
 	{Name: "name", Typ: types.String},
 	{Name: "status", Typ: types.String},
 }
+
+var TenantColumnsWithReplication = ResultColumns{
+	{Name: "id", Typ: types.Int},
+	{Name: "name", Typ: types.String},
+	{Name: "status", Typ: types.String},
+	{Name: "source_tenant_name", Typ: types.String},
+	{Name: "source_cluster_uri", Typ: types.String},
+	{Name: "replication_job_id", Typ: types.Int},
+	{Name: "latest_fully_replicated_time", Typ: types.Timestamp},
```

Comment:
Just noting based on our Observability discussion that we should rename the last two columns to be more user friendly. Some names that were thrown around were:

`replicated_time` and `retained_time`

Reply:
Done.